### DISCUSSION TITLE
Plugin support with AssemblyLoadContext

### DISCUSSION THREAD

**Discussion by truthz03** [elsa-workflows/elsa-core#5351] [Ideas]
Hi,

for my project I need support to dynamically load Plugins with an AssemblyLoadContext.
Everything is working now but I have some problems deserialising the payload from the store at startup because the payload type is not resolvable by `Type.GetType(typeName)` inside the `PolymorphicObjectConverter`.

I have already made some changes inside your code to fix this problem but I'm no able to create a PR.
Do I need some special permissions for that?

It would be really nice to get these changes into your code, otherwise I have to implement a big workaround for this.

Thanks
Thomas

**Comment by glime-ai** [comment]
Hello Thomas,

To contribute changes to the Elsa project, you typically need to fork the repository, make your changes in your fork, and then submit a pull request (PR) to the original repository. Here are the general steps you would follow:

1. Fork the Elsa repository on GitHub.
2. Clone your fork locally.
3. Create a new branch for your changes.
4. Make your changes in your new branch.
5. Commit your changes and push them to your fork on GitHub.
6. Go to the original Elsa repository on GitHub and create a new pull request.
7. Fill out the PR form with a clear description of your changes and why they are needed.
8. Submit the PR for review by the Elsa maintainers.

You do not need special permissions to submit a PR. However, your changes will be reviewed by the maintainers of the project before they can be merged into the main codebase.

If you are having trouble creating a PR, make sure you are logged into your GitHub account and that you have pushed your changes to your fork. If you are still experiencing issues, please provide more details about the problem you are encountering.

For more detailed instructions on contributing to Elsa, you can refer to the contribution guidelines provided in the Elsa documentation or the README file of the Elsa repository.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).

**Reply by truthz03** [reply]
I have created a PR now: https://github.com/elsa-workflows/elsa-core/pull/5352
